stores/nvidia.py

```diff
@@ -262,7 +280,11 @@ def buy(self, product_id):
 
     def is_in_stock(self, product_id):
         response = self.session.get(
-            NVIDIA_STOCK_API.format(product_id=product_id, locale=self.locale),
+            NVIDIA_STOCK_API.format(
+                product_id=product_id,
+                locale=self.locale,
+                currency=self.get_currency_for_locale(locale=self.locale),
```

Comment:
should this just be get_currency_for_locale(locale)?

Reply:
or ```CURRENCY_LOCALE_MAP.get(locale, "USD")```

Reply:
😄 That's what I had originally but it threw an exception.

Reply:
Thats because ```get_currency_for_locale()``` is a class function. If you had placed it outside of the class, then you can just call ```get_currency_for_locale(locale)```. But we can just get rid of the function entirely.

Reply:
Will update